### PR TITLE
py-gtdbtk and fastani: new packages

### PR DESCRIPTION
**Pull request by luke-dt**

### CODE REVIEW

File: var/spack/repos/builtin/packages/fastani/package.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/fastani/package.py]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

File: var/spack/repos/builtin/packages/py-gtdbtk/package.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-gtdbtk/package.py]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

**Review by tldahlgren — DISMISSED** [review]
Confirmed sha256.  Deferring merge decision until @adamjstewart has a chance to comment on `py-gtdbtk`.

**Review by tldahlgren — COMMENTED** [review]
Oops.  The copyright year has to be fixed first.

**Review by tldahlgren — DISMISSED** [review]

File: var/spack/repos/builtin/packages/py-gtdbtk/package.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    """GTDB-Tk is a software toolkit for assigning objective taxonomic
+       classifications to bacterial and archaeal genomes based on the Genome
+       Database Taxonomy (GTDB)."""
+
+    homepage = "https://github.com/Ecogenomics/GTDBTk"
+    pypi     = "gtdbtk/gtdbtk-2.1.0.tar.gz"
+
+    version('2.1.0', sha256='980885141f13502afdf05e720871427e3de4fe27f4f3f97e74af6fed87eb50a7')
+
+    depends_on('python@3.6:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-gtdbtk/package.py:20]: Needs a build dep on setuptools

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**